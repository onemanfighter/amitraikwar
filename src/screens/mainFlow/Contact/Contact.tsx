import { Box, Text } from '@chakra-ui/react';

const Contacts = () => {
  return (
    <Box>
      <Text>Contacts</Text>
    </Box>
  );
};

export default Contacts;
