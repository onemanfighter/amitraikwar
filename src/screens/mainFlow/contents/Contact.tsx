import { Box, Heading, HStack, Text, VStack } from '@chakra-ui/react';
import { useTranslation } from 'react-i18next';

const Contact = () => {
  const { t } = useTranslation();
  return (
    <VStack minH={'100vh'} id="contact" paddingTop={24}>
      <Box rowGap={10} width={'full'}>
        <Heading textAlign={'start'}>{t('contact.title')}</Heading>
        <HStack></HStack>
      </Box>
    </VStack>
  );
};

export default Contact;
