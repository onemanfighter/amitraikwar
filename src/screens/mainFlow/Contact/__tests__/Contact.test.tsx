import { render } from '@testing-library/react';
import Contact from '../Contact';

describe('Contact', () => {
  it('should render correctly', () => {
    const { container } = render(<Contact />);

    expect(container).toMatchSnapshot();
  });
});
