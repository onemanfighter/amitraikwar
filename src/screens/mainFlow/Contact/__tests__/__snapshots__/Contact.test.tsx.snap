// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render correctly 1`] = `
<div>
  <div
    class="css-0"
  >
    <p
      class="chakra-text css-0"
    >
      Contacts
    </p>
  </div>
</div>
`;
